routes/users.js

```diff
@@ -11,6 +11,7 @@ router.post("/verify", authenticate, users.verifyUser);
 router.get("/userId/:userId", users.getUserById);
 router.patch("/self", authenticate, userValidator.updateUser, users.updateSelf);
 router.get("/", authenticate, users.getUsers);
+router.get("/idle", users.getIdleUsers);
```

Comment:
Shouldn't we structure this endpoint using query params like `/users?status=idle|active`
It'll be more flexible this way

Reply:
In that case, we don't need a new endpoint. We will be enhancing https://github.com/Real-Dev-Squad/website-backend/blob/develop/routes/users.js#L13 this endpoint

Reply:
Change requirement as we discussed https://github.com/Real-Dev-Squad/website-backend/issues/635#issue-1318078384
Will work on it.